### ISSUE TITLE
Unexpected interaction between "orderby" argument and backwards pagination

### ISSUE THREAD

**Issue by ardiewen** [wp-graphql/wp-graphql#1411]
Hi all, wondering if this is intended behaviour or possibly a bug?

### Issue Summary

When paginating forward on an ordered post list (e.g. alphabetical by title) using `first` + `after` arguments, everything works as expected.

However, running the same query backwards, swapping `first` + `after` to `before` + `last` causes unexpected behaviour, assuming everything else is equal: **doesnt return the right posts, nor the right order.**

Changing the `order` argument to the opposite (e.g. "ASC" to "DESC") during backwards pagination seems to fix this issue, but is this the intended behaviour? Reference: [Relay GraphQL Cursor Connections Specification,  4.3 - Edge Order](https://relay.dev/graphql/connections.htm)

### Expected Behaviour

1. Specify a query with `{orderby: { field: "TITLE", order: "ASC" } }` and forward pagination arguments
2. Paginate in a forwards direction.
3. Change only the `first` + `after` arguments to `last` + `before`.
4. Paginate in a backwards direction and see the **exact same pages that forward pagination showed previously**.

### Actual Behaviour / Steps to Replicate

Let's say I have 13 posts with the following titles:
`['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M']`

1. Paginating in a forwards directions works as expected:

- first arguments are `first: 5`, `after: null`, `last: null`, `before: null`, `field: "TITLE"`, `order: "ASC"` and correctly returns posts **A, B, C, D, E,** and the appropriate **pageInfo**.

- second arguments are `first: 5`, `after: ${firstResult.pageInfo.endCursor}`, `last: null`, `before: null`, `field: "TITLE"`, `order: "ASC"` and correctly returns posts **F, G, H, I, J** and the appropriate **pageInfo**.

2. Paginating backwards doesn't work, or at least works in an unexpected way:

- third arguments (paginating backwards) are `first: null`, `after: null`, `last: 5`, `before: ${secondResult.pageInfo.startCursor}`, `field: "TITLE"`, `order: "ASC"` and incorrectly returns **K, J, I, H, G** with the corresponding **pageInfo**. The correct response here should have been the first page: **A, B, C, D, E**.

- fourth arguments (paginating backwards), `first: null`, `after: null`, `last: 5`, `before: ${secondResult.pageInfo.startCursor}`, `field: "TITLE"`, `order: "DESC"` returns the correct page and order of posts: **A, B, C, D, E**.

#### For reference, my query:
```
query GET_ALL_SORTED_POSTS ($first: Int, $last: Int, $after: String, $before: String, $orderByField: PostObjectsConnectionOrderbyEnum!, $order: OrderEnum!) {
  posts(after: $after, before: $before, first: $first, last: $last, where: {orderby: {field: $orderByField, order: $order}}) {
    edges {
      cursor
      node {
        __typename
        id
        title
      }
    }
    pageInfo {
      startCursor
      endCursor
      hasNextPage
      hasPreviousPage
    }
  }
}
```

#### Environmental Considerations
- Wordpress 5.2.7
- WPGraphQL 0.10.3
- All other plugins disabled
- Sanitized data environment (no other posts, pages, media, etc)
- Tested via client: WPGraphiQL 1.0.1
- Tested via client: Apollo Client 3.0.2 (React)

**Comment by bkstorm** [comment]
I have the same issue, I think it's a bug.

**Comment by jasonbahl** [comment]
#1773 addresses this. 

@ardiewen the information in this issue is fantastic and allowed me to replicate easily and write some tests to ensure this behavior continues to work in the future!

See the tests here where we create 26 posts titled A-Z, then paginate forward several pages and backward several pages using `{orderby: { field: "TITLE", order: "DESC" } }`

**Comment by jasonbahl** [comment]
Closed by #1773
